src/components/activity-page/managed-interactive/managed-interactive.tsx

```diff
@@ -1,12 +1,21 @@
 import React from "react";
 import { IframeRuntime } from "./iframe-runtime";
 import useResizeObserver from "@react-hook/resize-observer";
+import { Embeddable } from "../../../types";
 
 interface IProps {
-  embeddable: any;
+  embeddable: Embeddable;
   questionNumber?: number;
 }
 
+interface x {
+  foo: string;
+}
+
+interface y {
+  bar: string;
+}
+
```

Comment:
Probably meant to come out?

Reply:
Ha, good catch.

That was just me confirming to my own satisfaction that we can't do

```
interface x {
  foo: string;
}

interface y {
  bar: string;
}

const getFooBar = (xOrY: x | y) {
  if (xOrY.foo) {
    ....
  }
}
```

which is really annoying...

Reply:
I mean... that was a test to find out if anyone would really read through the whole commit.